rcl/test/rcl/test_arguments.cpp

```diff
@@ -966,6 +1059,10 @@ TEST_F(CLASSNAME(TestArgumentsFixture, RMW_IMPLEMENTATION), test_no_param_overri
   EXPECT_EQ(RCL_RET_INVALID_ARGUMENT, ret) << rcl_get_error_string().str;
   rcl_reset_error();
 
+  rcl_arguments_t empty_parsed_arg = rcl_get_zero_initialized_arguments();
+  ret = rcl_arguments_get_param_overrides(&empty_parsed_arg, &params);
+  EXPECT_EQ(RCL_RET_INVALID_ARGUMENT, ret) << rcl_get_error_string().str;
+
```

Comment:
@Blast545 nit: missing error state/reset?

Reply:
Added only a reset to match the rest of the old tests in the group, see cf35778